docs/app/templates/application.hbs

```diff
@@ -1,85 +1,78 @@
-{{#unless this.hideNav}}
```

Comment:
Good catch that `hideNav` could never be `true` as there isn't any route containing `'acceptance'`. :clap: